src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Program.cs

```diff
@@ -105,5 +139,16 @@ private static async Task<int> Main(string[] args)
 #endif
 
         return await rootCmd.InvokeAsync(args).ConfigureAwait(false);
+
+        void requiresPathOrEndpoint(CommandResult cmd)
```

Comment:
minor: Can we change to `RequiresPathOrEndpoint()`? Iirc nested functions we have created in other places start with capitals. I was a bit confused at the use site since I expected the lambda variable to be declared before use but then realized it is a nested function :)